js/cell.js

```diff
@@ -64,6 +68,10 @@ Cell.prototype.setPosition = function( x ) {
   this.renderPosition( x );
 };
 
+Cell.prototype.setAriaHidden = function ( val ) {
+  this.element.setAttribute('aria-hidden', val);
+};
```

Comment:
Adhering to project's code style is a nice way to partner into the project. I use spaces in parens, no space after `function`

``` js
Cell.prototype.setAriaHidden = function( val ) {
  this.element.setAttribute( 'aria-hidden', val );
};
```